.env.example

```
@@ -46,28 +9,51 @@ POSTGRES_PASSWORD="changeme"
+# PROXMOX_TOKEN_ID=""
+# PROXMOX_TOKEN_SECRET=""
+# PROXMOX_ALLOW_SELF_SIGNED="true"
 
 # Docker Stack Management
-# DOCKER_MANAGEMENT_FEATURE_FLAG="true"
-# GIT_REPOS_DIR="/data/repos"
-# GIT_SERVER_TOKEN="<random-token-here>"  # Required for git clone/push authentication
+DOCKER_MANAGEMENT_FEATURE_FLAG="true"
+VITE_DOCKER_MANAGEMENT_FEATURE_FLAG="true"
+GIT_REPOS_DIR="./data/repos"
+GIT_SERVER_TOKEN="dev-git-token"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)